pkgs/development/python-modules/colormath/default.nix

```
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, fetchPypi
+, networkx
+, numpy
+, lib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "colormath";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    inherit pname;
+    sha256 = "3d4605af344527da0e4f9f504fad7ddbebda35322c566a6c72e28edb1ff31217";
+  };
+
+  # Are there no tests?
+  doCheck = false;
+
+  propagatedBuildInputs = [ networkx numpy ];
+
+  meta = {
+    homepage = "https://github.com/gtaylor/python-colormath";
```

generally description comes first
```suggestion
    description = "Color math and conversion library";
    homepage = "https://github.com/gtaylor/python-colormath";
```